File: CarRentalApi/CarRentalApi/Models/Order.cs
using System;
using System.Collections.Generic;

namespace CarRentalApi.Models
{
    public partial class Order
    {
        public int Id { get; set; }
       
        public DateTime BeginDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Comment { get; set; }

        public int CarId { get; set; }
        public int UserId { get; set; }

        public  Car Car { get; set; }
        public  User User { get; set; }
    }
}

File: Client/ClientApp/src/app/orders/order.ts
export class Order{
    id: number;
    carId:number;
    userId:number;
    name:string;
    surname:string;
    brand:string;
    model:string;
    beginDate:string;
    endDate:string;
    comment:string;
}
export class UserCombo{
    id: number;
    name: string;
    surname:string;
    birth:string;
    licenseNumber:string;
}
export class CarCombo
{
    id: number;
    brand: string;
    model:string;
    class:string;
    year:number;
    registrationNumber:string;
   
}
export class OrderIns
{
    id:number;
    carId:number;
    userId:number;
    beginDate:string;
    endDate:string;
    comment:string;
}

